content/blog/2021/10/02/MonitorTheSolver.adoc

```diff
@@ -0,0 +1,215 @@
+= Monitor the solver
+Christopher-Chianelli
+2021-10-02
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: monitoring, production
```

Comment:
That grafana screenshot (especially the graph, the rest can be cut off) is a good candidate for the social media share image:

```suggestion
:jbake-tags: monitoring, production
:jbake-social_media_share_image: GrafanaGraphExample.png 
```

Reply:
Or the PrometheusUI.png image, that might be even better.

Reply:
the prometheus doesn't show an optaplanner metric, so the grafana one is better